components/org.wso2.carbon.business.rules.web/src/utils/BusinessRulesUtilityFunctions.js

```diff
@@ -212,57 +150,46 @@ class BusinessRulesUtilityFunctions {
         // Load available Business Rules
         let businessRulesPromise = BusinessRulesUtilityFunctions.getBusinessRules()
         businessRulesPromise.then(function (response) {
+            // render(<Route path='/' component={BusinessRulesManager}
```

Comment:
Any reason to keep this code segment commented?
Please check other places also..